src/store/reducers/cohorts.reducer.js

```diff
@@ -0,0 +1,14 @@
+import { COHORTS } from '../actions/actions.type';
+
+export const INITIAL_COHORTS_STATE = {
+  cohortApplicants: {}
+};
+
+export const cohortsReducer = (state = INITIAL_COHORTS_STATE, action = {}) => {
```

Comment:
why `INITIAL_COHORTS_STATE` and not just `INITIAL_STATE`?

Reply:
@martapa No particular reason :) I'll update that!